### PR TITLE
REVIEW 7 : ps unit customerhelpertest

### PR DESCRIPTION
**Pull request by ClaireAlma**
### Reason for change

[Linear task](https://linear.app/almapay/issue/ECOM-1577/[%F0%9F%A7%A9-ps]-unit-customerhelpertest)

### Code changes

image
### How to test

make coverage

### CODE REVIEW

File: alma/tests/Unit/Helper/CustomerHelperTest.php [hunk 1]
```diff
@@ -24,19 +24,104 @@
...
 
+use Alma\PrestaShop\Builders\CustomerHelperBuilder;
+use Alma\PrestaShop\Factories\ContextFactory;
+use Alma\PrestaShop\Helpers\OrderHelper;
+use Alma\PrestaShop\Helpers\ValidateHelper;
 use PHPUnit\Framework\TestCase;
 
 class CustomerHelperTest extends TestCase
 {
     public function testGetCustomer()
```
Comment (Benjamin-Freoua-Alma) [inline alma/tests/Unit/Helper/CustomerHelperTest.php:35]: Maybe we can split this test with different name for a better understanding of the test and a smaller function

Comment (ClaireAlma) [reply alma/tests/Unit/Helper/CustomerHelperTest.php:35]: I will comment the tests in the function

File: alma/tests/Unit/Helper/CustomerHelperTest.php [hunk 2]
```diff
@@ -24,19 +24,104 @@
...
+        $customer->passwd = '12345';
+        $customer->save();
+
+        $customerHelperBuilder = new CustomerHelperBuilder();
+        $customerHelper = $customerHelperBuilder->getInstance();
+
+        $this->assertInstanceOf(\Customer::class, $customerHelper->validateCustomer($customer));
     }
 
     public function testIsNewCustomer()
```
[the remark this answers is not in the document]
Comment (ClaireAlma) [reply alma/tests/Unit/Helper/CustomerHelperTest.php:107]: There is no needed for that, it will just complicated the comprehension

**Review by joyet-simon — APPROVED** [review]
Tests done

File: alma/tests/Unit/Helper/CustomerHelperTest.php (hunk 1 above)
Comment (ClaireAlma) [reply alma/tests/Unit/Helper/CustomerHelperTest.php:35]: Done

**Pull request merged**